### DISCUSSION TITLE
Can IPX be deployed in Cloudflare Workers?

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [unjs/community#18] [Q&A] [answered]
I tried to start a project with Hono + Workers + IPX. But appears IPX only works with Node or Bun.
It would be nice to see an example of this stack or at least IPX with Workers.

**Comment by pi0** [comment, marked as answer]
Sadly not (yet) because of a limitation of Sharp wasm that requires thread support which is not available on cloudflare workers. We might add an alternative ipx optimizer backends for limited support however it might not be as performant as hosting ipx on a vps or serverless target...

**Reply by juliomuhlbauer** [reply]
It would be really nice to use IPX with Cloudflare.
It is possible to deploy in Vercel or Netlify?
